katalog/alertmanager-operated/deploy.yml

```diff
@@ -4,9 +4,10 @@ metadata:
   labels:
     alertmanager: main
   name: main
-  namespace: monitoring
```

Comment:
Here the `namespace` key is needed otherwise the CRD fails validation and the deploy fails.